webservices/common/views.py

```diff
@@ -1,3 +1,4 @@
+import sys
```

Comment:
Looks like an extraneous `import` statement that's not used made it in here.

Reply:
Nice catch!